src/rules/indentation.js

```diff
@@ -16,41 +16,46 @@ var availableConfigs = {
 
 var errors = [];
 
-function test(parsedLocation, config, type) {
+function test(parsedLocation, configuration, type) {
     // location.column is 1 index based so, when we compare with the expected indentation we need to subtract 1
-  if (--parsedLocation.column !== config[type]) {
-    errors.push({message: 'Wrong indentation for "' + type + '", expected indentation level of ' + config[type] + ', but got ' + parsedLocation.column,
+  if (--parsedLocation.column !== configuration[type]) {
+    errors.push({message: 'Wrong indentation for "' + type + '", expected indentation level of ' + configuration[type] + ', but got ' + parsedLocation.column,
                  rule   : rule,
                  line   : parsedLocation.line});
   }
 }
 
+function testStep(step, language, configuration, mergedConfiguration) {
+  var keyword = step.keyword;
+  var stepType = _.findKey(language, function(values) { return values instanceof Array && values.indexOf(keyword) !== -1; });
+  stepType = stepType in configuration ? stepType : 'Step';
+  test(step.location, mergedConfiguration, stepType);
+}
+
 function indentation(parsedFile, unused, configuration) {
   var language = languageMapping[parsedFile.language];
+  var mergedConfiguration = _.merge(availableConfigs, configuration);
   errors = [];
-  configuration = _.merge(availableConfigs, configuration);
 
   // Check Feature indentation
-  test(parsedFile.location, configuration, 'Feature');
+  test(parsedFile.location, mergedConfiguration, 'Feature');
 
   if (parsedFile.background) {
     // Check Background indentation
-    test(parsedFile.background.location, configuration, 'Background');
+    test(parsedFile.background.location, mergedConfiguration, 'Background');
 
     // Check Background steps
     parsedFile.background.steps.forEach(function(step) {
-      test(step.location, configuration, 'Step');
+      testStep(step, language, configuration, mergedConfiguration);
```

Comment:
This was missed, made it so that any steps that were part of the background always followed the 'Step' configuration and ignored the per type config.